perf_test/blas/blas3/KokkosBlas3_common.hpp

```diff
@@ -156,10 +157,10 @@ typedef enum TEST {
 } test_e;
 
 static std::string test_e_str[TEST_N]{
-    "blas", "batched_serial", "batched_serial_blocked", "batched_serial_simd",
-    "batched_serial_simd_blocked", "batched_serial_compact_mkl", "batched_team",
-    "batched_team_blocked", "batched_team_vector",
-    "batched_team_vector_blocked", "batched_team_simd",
+    "blas", "batched", "batched_serial", "batched_serial_blocked",
```

Comment:
Same as above. Is this "batched" for testing square matrix with heuristic algorithm? Should we rename it to, perhaps, "batched_heuristic"?